src/components/SearchBar/SearchBar.tsx

```diff
@@ -1,159 +1,53 @@
 import * as React from "react";
 import bem from "../../utils/bem";
-import Select from "../Select/Select";
-import TextField from "../TextField/TextField";
-import { Button } from "../..";
-import { ButtonIconPositions, ButtonTypes } from "../Button/ButtonTypes";
-import { ButtonOptions } from "../Button/Button";
 
 export interface SearchBarProps {
-    searchBarId?: string;
+    /** ID that other components can cross reference for accessibility purposes */
+    id?: string;
+
+    /** blockName for use with BEM. See how to work with modifiers and BEM here: http://getbem.com/introduction/ */
     blockName?: string;
-    searchBarAriaLabel?: string;
-    searchBarAriaLabelledBy?: string;
-    buttonId?: string;
-    dropdownId?: string;
-    dropdownAriaLabel?: string;
-    options?: React.ReactNode;
-    selectedField?: string;
-    searchValue?: string;
-    placeHolder?: string;
-    searchButtonAttributes?: {};
-    placeholderText?: string;
-    textFieldAriaLabel?: string;
-    textFieldAttributes?: {};
-    // helperVariant enables a ResearchNow specific alternate.
-    // It should not be used in any other context.
-    helperVariant?: "ResearchNow";
-    hasError?: boolean;
-    errorMessage?: string;
-    selectBlurHandler?: (event: React.MouseEvent) => void;
-    selectChangeHandler?: (event: React.MouseEvent) => void;
-    searchSubmitHandler: (event: React.MouseEvent) => void;
-    searchChangeHandler?: (event: React.FormEvent) => void;
+
+    /** Populates aria-label on the form */
+    ariaLabel?: string;
+
+    /** Populates aria-labelledby on the form */
```

Comment:
With these comments, it would be nice to use it as guidance for engineers.  So like, something like "ID of the label for the search input" 

(Also would be a helpful reminder, I can never remember what the difference is between aria-label and aria-labelledby haha)

Reply:
This is the Metronome documentation for this component, which is pretty vague:

```
Is a form with attributes role="search" and aria-label or aria-labelledby that is meaningful for the template in which it is used, eg "Search Research Catalog" or "Search NYPL.org"
```

I figured that without additional context about the template the SearchBar is being used in, we probably can't give much guidance about what the `aria-labelledby` would be populated by, but I think we can still improve on the definitions by using the language from [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute) defining the difference between the two. What do you think?